src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,27 @@
+.calendar {
+  width: calendarWidth($calendar-item-width, $calendar-item-border, $gap);
+  list-style: none;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
These styles are better move to the class .day because it's inner styles and using them like this does not allow the block to be reused if needed.

Reply:
as i think this mix need me for positioning day block inside parent block calendar.